src/sentry/api/endpoints/issue_occurrence.py

```diff
@@ -0,0 +1,71 @@
+from django.conf import settings
+from rest_framework import serializers, status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import Endpoint
+from sentry.api.permissions import SuperuserPermission
+
+# from sentry.api.serializers import SimpleEventSerializer, serialize
+from sentry.eventstream.kafka import KafkaEventStream
+from sentry.utils import json
+
+
+class IssueOccurrenceSerializer(serializers.Serializer):
+    # id = serializers.IntegerField()
+    event_id = serializers.CharField()
+    fingerprint = serializers.ListField()
+    issue_title = serializers.CharField()
+    subtitle = serializers.CharField()
+    evidence_data = serializers.DictField()
+    evidence_display = serializers.ListField()
+    type = serializers.IntegerField()
+    detection_time = serializers.DateTimeField()
+
+
+class IssueOccurrenceEndpoint(Endpoint):
+    permission_classes = (SuperuserPermission,)
+    private: True
+
+    def post(self, request: Request) -> Response:
+        """
+        Write Issue occurrence data to a Kafka topic
+        `````````````````````````````````
+        :auth: required
+        """
+        event = request.data.pop("event")
+        occurrence = request.data
+
+        # this needs to be passed an Event or GroupEvent instance which obviously doesn't work here
+        # but not sure what to use instead. maybe can skip this since it's a temp internal thing?
+
+        # event_serializer = serialize(event, request.user, SimpleEventSerializer())
+        # if not event_serializer.is_valid():
+        #     return Response(event_serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+        # event = event_serializer.validated_data
+
+        occurrence["event_id"] = str(event["event_id"])
+
+        serializer = IssueOccurrenceSerializer(data=occurrence)
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+        occurrence = serializer.validated_data
+
+        eventstream = KafkaEventStream()
+        topic = settings.KAFKA_EVENTS  # needs to be changed, not sure what the new topic is called
```

Comment:
Does this new topic exist yet? I didn't see anything obvious in the settings.

Reply:
@rjo100 I forgot if we defined this yet

Reply:
Looks like it's in your PR Dan 😛  https://github.com/getsentry/sentry/blob/2201a82807fecef20c6603e35740f8a3edf927d4/src/sentry/conf/server.py#L2563

Reply:
Haha, nah, that's the topic for writing to snuba via eventstream. This topic is separate to that and is for receiving incoming occurrences

Reply:
Ah oops, I checked open PRs by both of you and that's the only one I found so I guess we don't have it yet. Is there much to adding it?

Reply:
https://github.com/getsentry/sentry/blob/15c9580a05c627b3f9c9197f58e21f349540c5f3/src/sentry/conf/server.py#L2503

is this it?

Reply:
Ahh I think it's this: https://github.com/getsentry/sentry/blob/9dea3b4080cf59c7f5fb95a78b6bd60ffa35ec99/src/sentry/conf/server.py#L2510